source/_components/ads.markdown

```diff
@@ -14,13 +14,15 @@ ha_category:
   - Light
   - Sensor
   - Switch
+  - Cover
 ha_release: "0.60"
 ha_iot_class: Local Push
 redirect_from:
   - /components/binary_sensor.ads/
   - /components/light.ads/
   - /components/sensor.ads/
   - /components/switch.ads/
+  - /components/cover.ads/
```

Comment:
This is not necessary and can be removed

Reply:
The whole redirect_from chapter?

Reply:
No only the `cover.ads`

Reply:
done